src/reports_api/models/project.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import Column, ForeignKey, Integer, Boolean, String, Float
+from sqlalchemy.orm import relationship
+
+from .code_table import CodeTable
+from .db import db
+
+
+class Project(db.Model, CodeTable):
```

Comment:
Project doesn't have to inherit CodeTable, instead set the base class as `BaseModel`.

Reply:
Updated